### PR TITLE
GC HashMap

### PR DESCRIPTION
**Pull request by baldersheim**
@havardpe PR

### CODE REVIEW

File: vespalib/src/tests/stllike/hash_test.cpp [hunk 1]
```diff
@@ -29,6 +29,14 @@ namespace {
     };
 }
 
+TEST("test that hashValue gives expected response")
```
Comment (havardpe) [inline vespalib/src/tests/stllike/hash_test.cpp]: why remove that hashValue tests? are these the tests that ensure we are Java compliant?

**Review by havardpe — COMMENTED** [review]
Java-compatible hash values are needed in ranking, if this hashValue is that hashValue, the tests for it should not be removed.

File: vespalib/src/tests/stllike/hash_test.cpp (hunk 1 above)
Comment (baldersheim) [reply vespalib/src/tests/stllike/hash_test.cpp]: I think I only moved them. As far as I can see they are here.

Comment (havardpe) [reply vespalib/src/tests/stllike/hash_test.cpp]: my bad. Too tired today

**Pull request merged**